src/pages/ThorChainLP/components/AddLiquidity/AddLiquidityInput.tsx

```diff
@@ -1515,116 +1502,111 @@ export const AddLiquidityInput: React.FC<AddLiquidityInputProps> = ({
     virtualRuneDepositAmountFiatUserCurrency,
   ])
 
+  const handleDepositSubmit = useCallback(() => {
+    return isUnsafeQuote ? setShouldShowWarningAcknowledgement(true) : handleSubmit()
+  }, [handleSubmit, isUnsafeQuote])
+
   if (!poolAsset || !runeAsset) return null
 
   return (
     <SlideTransition>
-      {renderHeader}
-      <Stack divider={divider} spacing={4} pb={4}>
-        {pairSelect}
-        <Stack>
-          <FormLabel mb={0} px={6} fontSize='sm'>
-            {translate('pools.depositAmounts')}
-          </FormLabel>
-          {!opportunityId && (
-            <LpType
-              assetId={poolAsset.assetId}
-              opportunityId={activeOpportunityId}
-              onAsymSideChange={handleAsymSideChange}
-            />
-          )}
-          {tradeAssetInputs}
+      <WarningAcknowledgement
+        message={`This deposit has high slippage (${bnOrZero(slippageDecimalPercentage)
+          .times(100)
+          .toFixed(2)
+          .toString()}%). Proceed with caution.`}
```

Comment:
should we add a translation for this?

Reply:
Actioned in https://github.com/shapeshift/web/pull/6647/commits/05c292f7035995dab341ddf89c2ad7e64e3e383b.